mappings/net/minecraft/network/ClientConnection.mapping

```diff
@@ -1,34 +1,68 @@
 CLASS none/eq net/minecraft/network/ClientConnection
+	CLASS none/eq$1
+		METHOD a createInstance ()Lio/netty/channel/nio/NioEventLoopGroup;
+		METHOD b createInstance ()Ljava/lang/Object;
+	CLASS none/eq$2
+		METHOD a createInstance ()Lio/netty/channel/epoll/EpollEventLoopGroup;
+		METHOD b createInstance ()Ljava/lang/Object;
+	CLASS none/eq$3
+		METHOD a createInstance ()Lio/netty/channel/local/LocalEventLoopGroup;
+		METHOD b createInstance ()Ljava/lang/Object;
 	CLASS none/eq$5
 		METHOD initChannel initChannel (Lio/netty/channel/Channel;)V
+			ARG 0 channel
 	CLASS none/eq$6
 		METHOD initChannel initChannel (Lio/netty/channel/Channel;)V
+			ARG 0 channel
+	FIELD c ATTR_KEY_PROTOCOL Lio/netty/util/AttributeKey;
+	FIELD d CLIENT_IO_GROUP Lnone/on;
+	FIELD e CLIENT_IO_GROUP_EPOLL Lnone/on;
+	FIELD f CLIENT_IO_GROUP_LOCAL Lnone/on;
 	FIELD g LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD h side Lnone/fm;
+	FIELD j lock Ljava/util/concurrent/locks/ReentrantReadWriteLock;
 	FIELD k channel Lio/netty/channel/Channel;
 	FIELD l address Ljava/net/SocketAddress;
 	FIELD m packetListener Lnone/ev;
 	FIELD n disconnectReason Lnone/fa;
+	FIELD o encrypted Z
+	FIELD p disconnected Z
 	METHOD <init> (Lnone/fm;)V
 		ARG 0 side
 	METHOD a setMinCompressedSize (I)V
 		ARG 0 value
+	METHOD a (Lio/netty/channel/ChannelHandlerContext;Lnone/fl;)V
+		ARG 0 ctx
```

Comment:
Why `ctx` when the rest of the mappings here refer to this param type as `context` ?

Related: All `ctx` names in this PR.